tests/test_dynunet.py

```diff
@@ -128,15 +128,6 @@ def test_script(self):
 @skip_if_no_cuda
 @skip_if_windows
 class TestDynUNetWithInstanceNorm3dNVFuser(unittest.TestCase):
-    def setUp(self):
-        try:
-            layer = InstanceNorm3dNVFuser(num_features=1, affine=False).to("cuda:0")
```

Comment:
Since this part is removed, related import can be removed (https://github.com/Project-MONAI/MONAI/pull/6399/files#diff-8f2debbe9c6aa7e8c0b4f65cf36df76bcc5097ce006af6638ae37fb14950ec5eL25)